showpass/Showpass-E2E-tests
# TA-52 verify order (31-0341-465e-a1a7-910944b23fff) API payload

**Description:**
The following existing transaction will be tested as part of this user story: ‘Order 1e-d67b-428a-9748-d638af16e987’

Steps:
- Login as 'user-packages@mailinator.com"'
- Navigate to https://beta.showpass.com/account/my-orders/31-0341-465e-a1a7-910944b23fff/
- Verify the API response payload

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-52

**How to test:**
Run the following test suite and ensure all specs pass
cypress/e2e/api-requests/transactions-api.cy.js

**Test execution results:**

<img width="1723" alt="TA-52-spec-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/035ac69b-c379-4698-8dd6-47ee9d6704bc">
